Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -339,25 +379,22 @@ void vtkMRMLSliceNode::SetOrientation(const char* orientation)
     {
     return;
     }
- if (!strcmp(orientation, "Axial"))
-    {
-    this->SetOrientationToAxial();
-    }
-  else if (!strcmp(orientation, "Sagittal"))
-    {
-    this->SetOrientationToSagittal();
-    }
-  else if (!strcmp(orientation, "Coronal"))
-    {
-    this->SetOrientationToCoronal();
-    }
-  else if (!strcmp(orientation, "Reformat"))
-    {
-    this->SetOrientationToReformat();
-    }
-  else
+
+  if (!strcmp(orientation, "Reformat"))
+   {
+   this->SetOrientationToReformat();
+   return;
+   }
+
+  this->SliceToRAS->DeepCopy(this->GetSliceOrientationPreset(orientation));
+
+  //SetOrientationString() and SetOrientationReference()
+  //behaviour will be mantained for backward compatibility
+  if(this->SliceToRAS)
```

Comment:
this can never happen (you've just called a method of this->SliceToRAS, so if it was NULL then the application has already crashed)